scripts/utils/models/cli_models.py

```diff
@@ -12,3 +13,14 @@ class TaskNamedArgs(Enum):
 class TaskFlagArgs(Enum):
     SERIALIZE = 'serialize'
     SAVE_CSV = 'saveCSV'
+    VISUALIZE = 'visualize'
+    SAVE_CLUSTERS = 'saveClusters'
```

Comment:
There is actually one more flag for serializing clustered graph, but I suggest to combine it with `--serialize` flag (see #24 )

Reply:
Will you fix it in the PR#24?